Server/Ressources/Database/product.csv

```diff
@@ -0,0 +1,2 @@
+name;price;description;available_quantity;delivery_date;image_url;id;brand;category_id
```

Comment:
Maybe we should add an "HT" price and a reference

Reply:
done